js/scenes/FTU.js

```diff
@@ -0,0 +1,153 @@
+import React, { Component, PropTypes } from 'react';
+import {
+  View,
+  Image,
+  Text,
+  Button,
+  Dimensions,
+  StyleSheet,
+  Platform,
+} from 'react-native';
+import ViewPager from 'react-native-viewpager';
+import ViewPagerIndicator from '../components/ViewPagerIndicator';
+import { connect } from 'react-redux';
+import { theme } from '../../config';
+
+export class FTU extends Component {
+  constructor(props) {
+    super(props);
+
+    this.navigator = this.props.navigator;
+
+    const PAGES = [
+      () => this.renderPage1(),
+      () => this.renderPage2(),
+    ];
+    const dataSource = new ViewPager.DataSource({
+      pageHasChanged: (p1, p2) => p1 !== p2,
+    });
+
+    this.state = {
+      dataSource: dataSource.cloneWithPages(PAGES),
+    };
+  }
+
+  render() {
+    return (
+      <Image
+        source={require('../images/ftu/background.jpg')}
+        resizeMode="cover"
+        style={styles.container}>
+        <View style={styles.scrim}>
+          <ViewPager
+            ref="viewPager"
+            autoPlay={false}
+            isLoop={false}
+            locked={true}
+            dataSource={this.state.dataSource}
+            renderPage={(render) => render()}
+            renderPageIndicator={() => <ViewPagerIndicator/>}
+            style={styles.viewPager}/>
+        </View>
+      </Image>
+    );
+  }
+
+  renderPage1() {
+    return (
+      <View style={styles.screen}>
+        <Text style={styles.title}>{`Hear the story of London's vibrant street art`.toUpperCase()}</Text>
+        <Text style={styles.text}>This month Project Magnet brings you a selection of amazing street art from around London.</Text>
+        <View style={styles.button}>
+          <View style={styles.border}>
+            <Button
+              title="GET STARTED"
+              accessibilityLabel="Continue to the next screen."
+              onPress={() => this.refs.viewPager.goToPage(1)}
+              color={Platform.OS === 'ios' ? 'white' : 'transparent'}/>
```

Comment:
So after our UX discussions of yesterday, let's not focus too much on design. All will change.
I agree we need a specific button. Let's open an issue for this.